src/processors/ComponentDefinition.js

```diff
@@ -86,6 +86,22 @@ class ComponentDefinition {
 		});
 	}
 
+	/**
+	 * utility function to extract js template strings, so that they can be passed into a template tag function
+	 */
+	static extractTemplateVariables(templateString) {
```

Comment:
extracting out the template variable parsing so we can use that in the ImportComponent logic. This is otherwise unchanged.